ui/apps/platform/src/Containers/Violations/Details/DeploymentDetails.tsx

```diff
@@ -130,6 +150,28 @@ const DeploymentDetails = ({ deployment }) => {
                     <FlexItem>
                         <SecurityContext deployment={relatedDeployment} />
                     </FlexItem>
+                    <FlexItem>
+                        <Title headingLevel="h3" className="pf-u-my-md">
+                            All Network Policies for {deploymentObj.namespace}
+                        </Title>
+                        <Divider component="div" />
+                    </FlexItem>
+                    <FlexItem>
+                        <Card isFlat data-testid="network-policy">
+                            <CardBody>
+                                {namespacePoliciesList?.length > 0 ? (
+                                    <List>
+                                        {namespacePoliciesList?.map((netpol: any) => (
+                                            // eslint-disable-next-line react/no-array-index-key
+                                            <ListItem key={netpol.id}>{netpol.name}</ListItem>
```

Comment:
1. Network policies should bey sorted ascending by name, by default.
2. It looks like the interface suggested in https://issues.redhat.com/browse/ROX-9760 has the following properties:
   - It is a single-column table with column title "Name".
   - The column seems to be sortable.
   - There's no round bullet next to each item.
   - Items are separated by horizontal lines.

In case you can't fulfill all points, it is ok to negotiate or offload to UI team, but please be explicit which once you'll not be able to achieve.

Reply:
Updated to a table component, I think it's closer to the mockup now. The only point I couldn't address was having the rows sortable.

Reply:
I'm fine without sorting. Please check with Zhenpeng and Boaz if they see it as a must.

Reply:
Please ask Zhenpeng which a specific PatternFly component or style he was referencing. We should not be doing any customization beyond options that are already available in PatternFly.

Reply:
Checked, thanks Van. He's fine with using the `<Table>` for the current task.